### PR TITLE
[NEW] General alert banner

### PR DESCRIPTION
**Pull request by ggazzo**
- [x] small version
- [x] actions
- [x] close
- [x] large version
- [x] deal with large texts
- [x] timer to close

@RocketChat/core 

### how to use

```javascript
alerts.open({html: '<b>Update your Rocket Chat.</b> New version available (v0.65)', action(){alert('hi')}}) 
// or
alerts.open({text: ' Update your Rocket Chat. New version available (v0.65)', action(){alert('hi')}}) 

const params = 
{ 
    title: 'insert your title' ,// bold
    text:'put some text here',
    html:'you can use html to',
    closable:  false, //default
    action: function() { /* some function to be executed on click */ },
    timer: null // time in ms, default none,
    icon: null, 
    modifiers:[...] // ex: ['large'] will set rc-alerts--large class
}
```

#### Desktop
image

#### Mobile
image

#### Desktop
image

#### Mobile
image

#### demo 
1qrfgg2boh

### CODE REVIEW

**Comment by sampaiodiego** [comment]
I think the `font-size` should be a bit smaller.. same size as channel name for example

**Comment by ggazzo** [comment]
@sampaiodiego changed the font size

File: packages/rocketchat-ui/client/views/app/alerts.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		this.action.call(this, e, instance.data.data);
+		alerts.close();
+	},
+	'click .js-close'() {
+		alerts.close();
+	}
+});
+
+Template.alerts.helpers({
+	isSafariIos: /iP(ad|hone|od).+Version\/[\d\.]+.*Safari/i.test(navigator.userAgent)
```
[the remark this answers is not in the document]
Comment (ggazzo) [reply packages/rocketchat-ui/client/views/app/alerts.js]: not yet... I think that helper should be a global one..

**Comment by raymondfeliz** [comment]
My apologies, I see that there is a section for "How to use" but im not exactly sure how to use this feature. Is there a place where I have to insert those lines? Im sorry if this is a silly question! :( 

I updated a test server of mine to 0.62.0 just to play with this specific feature, im extremely grateful that you added this!

**Comment by geekgonecrazy** [comment]
It's an internal API level feature.  It can be used from another package.  In the future it might be exposed in other ways

**Comment by raymondfeliz** [comment]
Is there perhaps a ste-by-step guide with an example  on how to use this feature?

I went ahead and started trying to figure it out but havent gotten far sadly.

I can query and auth against the API with

curl -H "Content-type:application/json" \
      http://localhost:3000/api/v1/login \
      -d '{ "username": "--username--", "password": "--password--" }'

From there I have an auth token and a user ID to execute other api calls, but I cant figure out how to call on yours. Im sure its something really simple that im not aware of. 

Im thinking of writing a quick little web app to interact with the API to change these messages from a gui once I figure out how the calls are made.

Thanks in advance!

**Comment by geekgonecrazy** [comment]
Its not exposed via the rest api.  Its an internal api.  Which means you would have to create a meteor package and compile it to use it.

I'm not sure on the plans, but a rest api endpoint my be created to allow using it that way.

**Pull request merged**